tests/gtest/datahandle_attribute_test.cpp

```diff
@@ -332,4 +332,39 @@ TEST_F(DatahandleAttributeTest, Attribute_Double_Data_Outside_Intersection_Verti
                 testing::ThrowsMessage<std::runtime_error>(testing::HasSubstr("Vertical window is out of vertical bounds")));
 }
 
+TEST_F(DatahandleAttributeTest, Attribute_Double_Different_Number_Of_Samples_To_Border_Per_Dimension) {
```

Comment:
Some ascii art here please.

Reply:
Okay, no need to draw, but I would still like to see a better description of what is tested for. I guess the issue is that I have no clue what you mean by border. Is border the outer cube?

Feel free to demonstrate concepts in 1 or 2 dimensions. I will be able to extend the idea to 3 or more dimensions.

Reply:
You find the comment I've added not to be enough for understanding the test point?

(And yes, it is border of outer cube).